tex/LaTeXML/Portfile

```diff
@@ -38,6 +38,7 @@ depends_lib-append \
         port:p${perl5.major}-json-xs \
         port:p${perl5.major}-libwww-perl \
         port:p${perl5.major}-parse-recdescent \
+        port:p${perl5.major}-pod-parser \
```

Comment:
since you are adding a `lib` dependency you will need to increase the `revision` field to make sure it will be picked up by everyone who was the port already installed

Reply:
@reneeotten you're asking for those changes in the *commit* message, not the *PR* messages, right?  (I'm used to the latter for github, but I suppose since macports is using a hybrid github/trac setup...)

Reply:
The text or title of a PR have no impact on the commit history; indeed closing Trac tickets automatically will only work when the line I mentioned -with the full URL- is included in the body of the commit message. So yes, as I said in my initial comments the requested changes are for the commit message ;)